### PR TITLE
Distributor guide

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

**Review by tamtamchik — APPROVED** [review]
LGTM

**Review by mkurayan — APPROVED** [review]

**Review by eddort — APPROVED** [review]

File: docs/guides/reward-distributor-bot.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Reward distribution bot
+
+## Introduction
+
+Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on [Node Operator Registry](https://docs.lido.fi/contracts/node-operators-registry) smart contract.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:5]:
```suggestion
Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on the [Node Operator Registry](/contracts/node-operators-registry) smart contract. 
```

File: docs/guides/reward-distributor-bot.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Reward distribution bot
+
+## Introduction
+
+Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on [Node Operator Registry](https://docs.lido.fi/contracts/node-operators-registry) smart contract. 
+After [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate the reward distribution to allocate rewards among Node Operators in the Staking Module.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:6]:
```suggestion
After [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate the reward distribution to allocate rewards among Node Operators in the Staking Module unless the next oracle frame starts.
```

File: docs/guides/reward-distributor-bot.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:15]: Does it need a node or just an RPC endpoint? Light node, full node, archive node?

File: docs/guides/reward-distributor-bot.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Every epoch daemon checks the staking modules provided in environment. If a module has a non-distributed rewards, the bot pulls the trigger and distributes rewards between Node Operators inside this module.
+
+Basically bot is watching [RewardDistributionState](https://docs.lido.fi/contracts/node-operators-registry#getrewarddistributionstate) of module. 
+If module has `ReadyForDistribution` state, bot triggers [distributeReward](https://docs.lido.fi/contracts/node-operators-registry#distributereward) method to distribute rewards.
+
+### Envs
+
+Required variables are:
+- WEB3_RPC_ENDPOINTS - list of EL rpc endpoints separated by `,`.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:29]: why need a list?

**Review by TheDZhon — CHANGES_REQUESTED** [review]
Looks good yet some little improvements would be great to have :pray:

File: docs/guides/reward-distributor-bot.md (hunk 2 above)
Comment (F4ever) [reply docs/guides/reward-distributor-bot.md:6]:
```suggestion
After the [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate reward distribution to allocate rewards among Node Operators in the Staking Module, unless the oracle sends the next frame report.
```

File: docs/guides/reward-distributor-bot.md (hunk 3 above)
Comment (F4ever) [reply docs/guides/reward-distributor-bot.md:15]: Whatever, no special requirements

File: docs/guides/reward-distributor-bot.md (hunk 4 above)
Comment (F4ever) [reply docs/guides/reward-distributor-bot.md:29]:
```suggestion
- WEB3_RPC_ENDPOINTS - list of EL rpc endpoints separated by `,`. All except the first rpc will serve as fallback options.
```

File: docs/guides/reward-distributor-bot.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+### Nodes
+
+- Execution Node
+
+## How to use
+
+Every epoch daemon checks the staking modules provided in environment. If a module has a non-distributed rewards, the bot pulls the trigger and distributes rewards between Node Operators inside this module.
+
+Basically bot is watching [RewardDistributionState](https://docs.lido.fi/contracts/node-operators-registry#getrewarddistributionstate) of module.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:23]:
```suggestion
Basically bot is watching [RewardDistributionState](/contracts/node-operators-registry#getrewarddistributionstate) of module. 
```

why it is important: internal links are verified upon building stage and broken links are visible in CI

File: docs/guides/reward-distributor-bot.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
+# Reward distribution bot
+
+## Introduction
+
+Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on the [Node Operator Registry](/contracts/node-operators-registry) smart contract. 
+After the [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate reward distribution to allocate rewards among Node Operators in the Staking Module, unless the oracle sends the next frame report.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:6]:
```suggestion
After the [Accounting Oracle](/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate reward distribution to allocate rewards among Node Operators in the Staking Module, unless the oracle sends the next frame report.
```

File: docs/guides/reward-distributor-bot.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+### Nodes
+
+- Execution Node
+
+## How to use
+
+Every epoch daemon checks the staking modules provided in environment. If a module has a non-distributed rewards, the bot pulls the trigger and distributes rewards between Node Operators inside this module.
+
+Basically bot is watching [RewardDistributionState](https://docs.lido.fi/contracts/node-operators-registry#getrewarddistributionstate) of module. 
+If module has `ReadyForDistribution` state, bot triggers [distributeReward](https://docs.lido.fi/contracts/node-operators-registry#distributereward) method to distribute rewards.
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:24]:
```suggestion
If module has `ReadyForDistribution` state, bot triggers [distributeReward](/contracts/node-operators-registry#distributereward) method to distribute rewards.
```

File: docs/guides/reward-distributor-bot.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+### Envs
+
+Required variables are:
+- WEB3_RPC_ENDPOINTS - list of EL rpc endpoints separated by `,`. All except the first rpc will serve as fallback options.
+- NODE_OPERATOR_REGISTRY_ADDRESSES - List of staking modules that can accept rewards distribution. All addresses could be found [here](https://docs.lido.fi/deployed-contracts/).
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:30]:
```suggestion
- NODE_OPERATOR_REGISTRY_ADDRESSES - List of staking modules that can accept rewards distribution. All addresses could be found [here](/deployed-contracts/).
```

File: docs/guides/reward-distributor-bot.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
+
+- Execution Node
```
Comment (TheDZhon) [inline docs/guides/reward-distributor-bot.md:17]:
```suggestion
- Execution Node (can be an RPC)
```

**Review by TheDZhon — APPROVED** [review]
LGTM except only for a few suggestions

**Review by kadmil — APPROVED** [review]
LGTM

**Pull request merged**